src/components/select/examples/select-with-icons.tsx

```diff
@@ -0,0 +1,72 @@
+import { Option } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+/**
+ * Select with icons for options
+ */
+@Component({
+    shadow: true,
+    tag: 'limel-example-select-with-icons',
+})
+export class SelectExample {
+    @State()
+    public value: Option;
+
+    @State()
+    public disabled = false;
+
+    private options: Option[] = [
+        {
+            text: 'Spider-man',
+            value: 'spider',
+            icon: 'spiderman_head',
+            iconColor: 'rgb(var(--color-red-default))',
+        },
+        {
+            text: 'Iron Man',
+            value: 'iron',
+            disabled: true,
+            icon: 'iron_man',
+            iconColor: 'rgb(var(--color-coral-default))',
+        },
+        {
+            text: 'Batman',
+            value: 'bat',
+            icon: 'batman_old',
+            iconColor: 'rgb(var(--color-black))',
+        },
+        {
+            text: 'Superman',
+            value: 'super',
+            icon: 'superman',
+            iconColor: 'rgb(var(--color-blue-default))',
+        },
+    ];
+
+    constructor() {
+        this.onChange = this.onChange.bind(this);
+        this.toggleEnabled = this.toggleEnabled.bind(this);
+    }
+
+    public render() {
+        return (
+            <section>
+                <limel-select
+                    label="Favorite hero"
+                    helperText="If you don't see a heroine, it's our icon-provider's fault"
```

Comment:
😂👌

Reply:
😅

Reply:
I found Wonder Woman at least, so I added her and tweaked the helper-text 🤷‍♂️

Reply:
`helperText="If you see just one heroine, it's our icon-provider's fault"` ?
😄

Reply:
Oh I see what you wrote. Nice! 
Maybe the new icons8 has some up to date icons 😄 we could probably also use a cat icon and pretend it's Catwoman :P

Reply:
Yeah, I went with "lack of diversity", because they're not just men by overwhelming majority, they're also all white… 🙄

Reply:
Totally right! 👌 
This forces us to reuse the same cat icon and call it Black Panther then 😸 and change the helper text to:
`If you see a lack of diversity, it's both our icon-provider's and Hollywood's fault`

Reply:
I don't think Catwoman and Black Panther have any specific sign or icon.